README.md

```diff
@@ -46,15 +46,15 @@ Below are the various ways of generating images:
   Scala version [2.13.15] : 2.13.15
   Java version [11.0.25-amzn] : 11.0.25-amzn
   Play-Slick version [5.3.1] :
-  Sbt version [1.10.4] :
+  Sbt version [1.10.5] :
   Docker registry [ivanoronee] :
   [info] Working with versions:
   [info] - base-image => debian:bullseye-20241016-slim
```

Comment:
There's a new debian release yesterday. Let's update this as well to https://hub.docker.com/layers/library/debian/bullseye-20241111-slim/images/sha256-0ee558f46e262c21f9e3449896c59d740eaa323e3c0709f5c6f3c3943bd0a113?context=explore